exporter/sumologicexporter/factory.go

```diff
@@ -17,6 +17,14 @@ import (
 	"github.com/open-telemetry/opentelemetry-collector-contrib/exporter/sumologicexporter/internal/metadata"
 )
 
+const (
+	// The value of "type" key in configuration.
+	typeStr        = "sumologic"
+	stabilityLevel = component.StabilityLevelBeta
```

Comment:
These consts should be removed, you should use the generated metadata code from `internal/metadata/generated_status.go`: `metadata.Type`, `metadata.*Stability`.

Reply:
Done